docs/DeveloperGuide.md

```diff
@@ -425,39 +425,49 @@ Additionally, there is a static final static message to be displayed to the user
 
 ### Find feature
 
-#### Current Implementation
+The find mechanism implements the following sequence for the method call execute("find").
 
-The activity diagram below illustrates the flow of a `find` command.
+#### What is the find feature
 
-![FindFeatureActivityDiagram](images/FindFeatureActivityDiagram.png)
+The find feature allows users to find a particular contact in Tracey and retrieve their specific details.
+
+The `find` command is as follows:
+
+`find NAME`
 
-#### Usage Scenario
+The user do not require to search the full name of the contact. However, the name must be its substring.
```

Comment:
Can add the query substring must match the person's name starting from the first character eg. `kev` will match `kevin` but `vin` will not